prime-angular-frontend/src/app/modules/enrolment/pages/regulatory/regulatory.component.ts

```diff
@@ -87,6 +89,11 @@ export class RegulatoryComponent extends BaseEnrolmentProfilePage implements OnI
   public ngOnInit() {
     this.createFormInstance();
     this.patchForm().subscribe(() => this.initForm());
+
+    this.form.valueChanges.subscribe(() => {
+      this.shouldRemoveRemoteAccess = this.hasRemoteAccess && !this.isInitialEnrolment &&
```

Comment:
I'm probably missing something, or maybe my comment wasn't well worded, but you shouldn't need a new member variable to track the changes.  My comment was indicating that `hasRemoteAccess` is being orphaned after it is initially set, and that it can be more thoroughly used:

1. Having `this.hasRemoteAccess = this.canRequestRemoteAccess();` set in `initForm` and `subscribe`  so it is being reused
2. Make `hasRemoteAccess` as a public member variable for use in the template, and
3. In the template use *ngIf="!hasRemoteAccess && !initialEnrolment" so changes in the value are reflected in the UI

Let me know if this doesn't work for some reason, and maybe you can catch me up on what I'm not seeing.

Reply:
`hasRemoteAccess ` is not orphaned but used to store the initial status of enrolle's "have remote access". If you subscribe it, the value will be overwritten and the ngIf logic is no longer correct. 

For example, 
1. enrollee has CPBC pharmacist licence type hence no remote access. 
2. Hitting the page to modify licence type,  `hasRemoteAccess ` is initialized to false.  and ngIf evaluated to be true. The warning message will be displayed. This is not expected. I don't think we need to remind enrollee in this case.